src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/DescriptorReferenceValidation.feature.disabled

```diff
@@ -0,0 +1,120 @@
+             When client sends a POST request to "/ed-fi/localEducationAgencies" with body
+                  """
+                  {
+                    "localEducationAgencyId": 25590100,
+                    "nameOfInstitution": "Grand Institution Test",
+                    "categories": [
+                        {
+                            "educationOrganizationCategoryDescriptor": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor#Education+Service+Center"
+                        }
+                    ],
+                    "localEducationAgencyCategoryDescriptor": "uri://ed-fi.org/LocalEducationAgencyCategoryDescriptor#Public+school+district+part+of+a+supervisory+union"
+                  }
+                  """
+             Then it should respond with 201
+
+
+        @ignore
+        Scenario: User can UPDATE a resource when descriptor exists
+             When client sends a PUT request to "/ed-fi/students/{id}" with body
+                  """
+                  {
+                    "id": "{id}"
+                    "studentUniqueId": "604824",
+                    "birthDate": "2010-01-13",
+                    "firstName": "Traci",
+                    "lastSurname": "Mathews",
+                    "citizenshipStatusDescriptor": "uri://ed-fi.org/CitizenshipStatusDescriptor#US+Citizen"
+                  }
+                  """
+             Then it should respond with 204
+
+
+        @ignore
+        Scenario: User CANNOT update a resource using a descriptor that does not exist
+             When client sends a PUT request to "/ed-fi/students/{id}" with body
+                  """
+                  {
+                    "id": "{id}"
+                    "studentUniqueId": "123456789",
+                    "birthDate": "2010-03-25",
+                    "firstName": "Jeff",
+                    "lastSurname": "Test"
+                    "citizenshipStatusDescriptor": "uri://ed-fi.org/CitizenshipStatusDescriptor#Fake",
+                  """
+             Then it should respond with 400
+              And the response body is
+                  """
+                  {
+                    "detail": "Data validation failed. See 'validationErrors' for details.",
+                    "type": "urn:ed-fi:api:bad-request:data",
+                    "title": "Data Validation Failed",
+                    "status": 400,
+                    "correlationId": null,
+                    "validationErrors": {
+                        "$.citizenshipStatusDescriptor": [
+                            "CitizenshipStatusDescriptor value 'uri://ed-fi.org/CitizenshipStatusDescriptor#Fake' does not exist."
+                        ]
+                    }
+                  }
+                  """
+
+        @ignore
+        Scenario: User CANNOT create (upsert) a resource using a descriptor that does not exist
+          When client sends a POST request to "/ed-fi/students" with body
```

Comment:
Can we replace this to
`When a XXXXXXX request is made to "XXXXXXX" with`

Reply:
Done